stock_move_line_secondary_done/models/stock_move_line.py

```diff
@@ -52,3 +53,16 @@ def _onchange_qty_done(self):
             != 0
         ):
             self.secondary_uom_qty_done = qty
+
+    def _action_done(self):
+        res = super()._action_done()
+        move_lines = self.filtered(
+            lambda l: l.secondary_uom_id
+            and float_is_zero(
+                l.secondary_uom_qty_done,
+                precision_rounding=l.secondary_uom_id.uom_id.rounding,
+            )
```

Comment:
I think we can remove this condition to ensure the data consistency. I consider the performance loss to be negligible.
```suggestion
```